sys/net/link_layer/iQueue_MAC/Makefile

```diff
@@ -0,0 +1,8 @@
+MODULE = simple
```

Comment:
Is there a specific reason for this name? Otherwise you could just omit the line and the module would be automatically named `IQUEUE_MAC` which is IMO more speaking.